Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -611,6 +611,35 @@ def condition(self, fragment, stepList, key, hasHarvest):
     suffix = '_seedingDeepCore',
     offset = 0.17,
 )
+# pixel digiMorphing workflows
+class UpgradeWorkflow_siPixelDigiMorphing(UpgradeWorkflow):
+    def setup_(self, step, stepName, stepDict, k, properties):
+        if 'Reco' in step:
+            stepDict[stepName][k] = merge([{'--procModifiers': 'siPixelDigiMorphing'}, stepDict[step][k]])
+    def condition(self, fragment, stepList, key, hasHarvest):
+        result = (fragment=="QCD_Pt_1800_2400_14" or fragment=="TTbar_14TeV" ) and any(y in key for y in ['2022','2023','2024','2025'])
+        return result
+upgradeWFs['siPixelDigiMorphing'] = UpgradeWorkflow_siPixelDigiMorphing(
+    steps = [
+        'Reco',
+        'RecoFakeHLT',
+        'RecoGlobal',
+        'RecoGlobalFakeHLT',
+        'RecoNano',
+        'RecoNanoFakeHLT',
+    ],
+    PU = [
+        'Reco',
+        'RecoFakeHLT',
+        'RecoGlobal',
+        'RecoGlobalFakeHLT',
+        'RecoNano',
+        'RecoNanoFakeHLT',
+    ],
+    suffix = '_siPixelDigiMorphing',
+    offset = 0.9999,
```

Comment:
TBH, I am not sure if there's a policy about these offsets, but if there is it would be good to get a suggestion.

Reply:
The problem is that a policy has never been really defined and now if we want to have one it would mean (probably) changing many of them. That is not a problem in general for me, it only need some work. For this specific case, `*.5XX` wfs are usually dedicated to pixel related workflows, as far as I can tell.

Reply:
> For this specific case, *.5XX wfs are usually dedicated to pixel related workflows, as far as I can tell.

There is also

```
0.511: Legacy ECAL reco only (CPU)
0.521: Legacy HCAL reco only (CPU)
```

:)

Reply:
Then we are really doomed : (

Jokes apart maybe at this point something as `0.1X` (like `0.18`) could vaguely make sense. There are few tracking related wfs with that kind of offsets. But there also few others not related (`Neutron`, `JMENano`, `ecalDeepSC` ...), so it's really a mild preference.